packages/web-components/fast-components/src/skeleton/skeleton.definition.ts

```diff
@@ -11,28 +11,34 @@ export const fastSkeletonDefinitions: WebComponentDefinition = {
             attributes: [
                 {
                     name: "fill",
-                    description: "The fill attribute",
+                    title: "Fill",
+                    description: "Indicates the Skeleton should have a filled style",
                     type: DataType.string,
```

Comment:
Should this be a bool?

```suggestion
                    type: DataType.boolean,
```